src/components/Company/Offers/Filters/Location.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
```

Comment:
Maybe it would be nicer if the search was case-insensitive

Reply:
Done. I've added a prop that controls if it is or not to each filter, in order to not change the behavior in other components.